core/src/stores/migrations/20230824_data_sources_documents_source_parents

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE
+    data_sources_documents
+ADD
+    COLUMN parents text[] NOT NULL DEFAULT '{}';
```

Comment:
Isn't `DEFAULT ARRAY[]::text[]` what we want?

Reply:
Yes, it populates the field.  AFAICT '{}' is postgres for "empty array" so it would be OK but I agree expliciting the type is cleaner => I have changed this as you suggest